src/Resources/views/storefront/base.html.twig

```diff
@@ -1,9 +1,16 @@
 {% sw_extends '@Parent/storefront/base.html.twig' %}
+{% set current_currency_code = context.currency.isoCode %}
+
+{% if current_currency_code == 'GBP' %}
```

Comment:
what if someone uses different country codes than `de-DE` or `en-GB` ?

Reply:
As I red in docs "Most of the time de-DE and en-GB are covering everything needed." so I thought that we can use these 2 codes because this codes are describe the place of currency sign in the price